### ISSUE TITLE
learning rate schedule

### ISSUE THREAD

**Issue by ink1** [NVlabs/stylegan2-ada#86]
hey everyone.
iirc, the old StyleGAN had a learning rate schedule which progressively reguced LR in stages once reaching certain numbers of kimg.
I've seen it is recommended to start from 15000 kimg for this one when doing transfer learning from a pre-trained model.
However I'd like to find out about all the stages.
Much appreciated if anyone can point me to the relevant code or tell what the stages are.
